apps/wallet/src/ui/app/components/accounts/AccountItem.tsx

```diff
@@ -44,6 +45,8 @@ export const AccountItem = forwardRef<HTMLDivElement, AccountItemProps>(
 		},
 		ref,
 	) => {
+		const { data: accounts } = useAccounts();
+		const account = accounts?.find((account) => account.address === address);
```

Comment:
We should change this component to accept the account id. There is a case where we can have more than one accounts with the same address (edge case but possible). This is fine for now we can change it later